### ISSUE TITLE
False positive for valid role=image

### ISSUE THREAD

**Issue by scottaohara** [validator/validator#1771]
per the following test
```
<div role=image>foo</div>
```

the checker reports a bad ARIA role.  but this is a [valid role](https://w3c.github.io/aria/#image) which is supported by all browsers ([wpt demonstrating role=image supported in all browsers](https://wpt.fyi/results/wai-aria/role/synonym-roles.html?label=pr_head&max-count=1&pr=42203).

The checker needs to be updated to remove the false positive for instances of this valid `image` role.
